app/views/spree/admin/products/new.html.haml

```diff
@@ -25,16 +31,19 @@
           = f.field_container :variant_unit do
             = f.label :variant_unit, t(".units")
             %span.required *
-            = f.select 'variant_unit', [],
-              { include_blank: true },
-              { id: 'product_variant_unit_with_scale',
-              name: 'product_variant_unit_with_scale',
-              'ng-model' => 'product.variant_unit_with_scale', 
-              'ng-options' => 'unit[1] as unit[0] for unit in variant_unit_options', 
-              "data-controller": "tom-select", 
-              "data-tom-select-options-value": '{"allowEmptyOption":false}', 
-              class: "primary",
-              }
+            = render(SearchableDropdownComponent.new(form: f,
+                name: :variant_unit,
+                aria_label: t('.units'),
+                options: [],
+                selected_option: @product.variant_unit,
+                include_blank: true,
+                placeholder_value: t('.search_for_units'),
+                other_attrs: { id: 'product_variant_unit_with_scale',
+                name: 'product_variant_unit_with_scale',
+                'ng-model' => 'product.variant_unit_with_scale', 
+                'ng-options' => 'unit[1] as unit[0] for unit in variant_unit_options', 
+                "data-controller": "tom-select",
```

Comment:
This is already included in `SearchableDropdownComponent` , could you remove it ?

Reply:
Actually the variant_unit dropdown only has a short predefined list of options, and so doesn't need the 'searchable' feature anyway. So I suggest `variant_unit` is not changed. (as a bonus this will avoid a conflict with the Product Refactor PR). 
Could you remove this change please?